src/wp-includes/script-loader.php

```diff
@@ -279,6 +279,15 @@ function wp_default_packages_scripts( $scripts ) {
 	 */
 	$assets = include ABSPATH . WPINC . "/assets/script-loader-packages{$suffix}.php";
 
+	// Add the private version of the Interactivity API manually.
+	$scripts->add(
+		'wp-interactivity',
+		'/wp-includes/js/dist/interactivity.min.js',
+		array(),
+		'6.4.0',
+		1
```

Comment:
@felixarntz, does this script need to define the `defer` strategy to ensure that view scripts depending on it don't fall back to the legacy script loading. They should all use the defer strategy after changes enabled by @westonruter.

Reply:
@gziolo Indeed, that would be great!

And we don't need to worry about breakage either, since if a script that depends on it doesn't use `defer`, it would still be forced back as well.

Reply:
Yes, the `wp-interactivity` script was intended to be loaded in the `head` with `defer`. See `gutenberg_interactivity_move_interactive_scripts_to_the_footer()` in Gutenberg: https://github.com/WordPress/gutenberg/pull/52536/files#diff-2cfd8de00ffba84501209430457fc075fcffcd8fdc45bdcb0c41cfd2b28e7b2cL14-L26